lib/components/Card/card.css

```diff
@@ -0,0 +1,264 @@
+.card-component {
+    display: flex;
+    flex-direction: column;
+    gap: 0.5rem;
+}
+
+.card-grid {
+    display: grid;
+    gap: 2rem;
+    margin: 1.25rem;
+    justify-content: center;
+    align-items: center;
+    grid-template-areas:
+        "card-basic basic-iframe"
+        "card-horizontal horizontal-iframe"
+        "card-vertical vertical-iframe"
+        "card-ver-dismiss ver-dismiss-iframe"
+        "card-ver-overlay ver-overlay-iframe";
+}
+
+.card {
+    border: 1px solid var(--secondary-color);
+    box-shadow: var(--shadow-color) 0px 3px 8px;
+    position: relative;
+    border-radius: 0.5rem;
```

Comment:
if value is less than 10px you can use px, but its upto you.

Reply:
Have changed it to px.